Update create-expo-app CLI help screen and README to reflect new default template in v51 (#28738)
expo/expo: packages/create-expo/README.md

## Templates

> Create Expo App prepares the app with the `tabs` template by default. You can choose to use a project with more prepared functionality. For that, you can start Create Expo App with the `--template` flag. Use the `--help` flag to see available template options.

Reviewer: ```suggestion
Create Expo App prepares the app with the `default` template. You can run Create Expo App with the `--template` flag to choose from other templates, or with the `--example` flag to choose from [example projects](https://github.com/expo/examples). Use the `--help` flag to see available template options.
```
